src/main/scala/com/typesafe/sbt/packager/archetypes/JavaAppKeys.scala

```diff
@@ -26,4 +26,6 @@ trait JavaAppKeys {
     "scriptClasspath",
     "A list of relative filenames (to the lib/ folder in the distribution) of what to include on the classpath."
   )
+  val bundledJvmLocation =
```

Comment:
This is a binary incompatible change (see build error https://travis-ci.org/sbt/sbt-native-packager/jobs/527261199#L1165).

This can either be fixed by adding a line to the `build.sbt` (here https://github.com/sbt/sbt-native-packager/blob/master/build.sbt#L60) or by a new, private trait that is being extended in the `autoImport` object. I favour the second option.

Reply:
I've done this (added a `JavaAppKeys2` private trait). However there are other bincompat-breaking changes since I had to add bundled JVM defines to the script plugins. I can handle `BashStartScriptPlugin.Defines.apply` (just add an overload and delegate to it), but I had to change the `BatStartScriptPlugin.BatScriptConfig` - and it looks like working around this one would just bring more breaking changes in the future...

Reply:
I've worked around other MiMa problems in d6b4b94 just in case - but I still think that it would be better to make a minor release with this.